config/permissions.php

```diff
@@ -81,7 +81,7 @@
     'manager' => array(
         '/projectsEvaluation.php', '/usersEvaluation.php', '/projectsSummary.php', '/services/getFilteredCustomProjectsService.php',
         '/services/getUserStoryReportJsonService.php', '/services/getProjectCustomerReportJsonService.php',
-        '/services/getProjectUserReportJsonService.php',
+        '/services/getProjectUserReportJsonService.php', '/services/getCustomerProjectsService.php',
```

Comment:
`getCustomerProjectsService` is already listed in `staff`, you don't need it here.